### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LordKobra**
* small adjustments to grammar and wording

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,47 +30,47 @@ Supported systems are:
...
    or you can edit `Gothic2Notr.bat` and run this file instead.
 
 ### Linux
-1. If not already done install Gothic via Wine/Proton or copy the game files from a Windows installation.
-2. You can download a build from [CI](https://ci.appveyor.com/project/Try/opengothic/history?branch=master) and extract into a folder of your choice. Alternatively OpenGothic can be built manually. For Arch the [AUR](https://aur.archlinux.org/packages/opengothic) provides a 3rd party package.
+1. If not already done, install Gothic 2 via Wine/Proton or copy the game files from a Windows installation.
+2. You can download a build from [CI](https://ci.appveyor.com/project/Try/opengothic/history?branch=master) and extract into a folder of your choice. Alternatively, OpenGothic can be built manually. For Arch the [AUR](https://aur.archlinux.org/packages/opengothic) provides a third-party package.
 
-3. Run `Gothic2Notr.sh -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"` (example path, use path to your Gothic installation instead)
+3. Run `Gothic2Notr.sh -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"` (example path, use the path to your Gothic 2 installation instead)
```
Comment (Try) [inline README.md:52]: I'm not sure about `the path`. While I'm not great at English, `the` should be something very concrete and "path to gothic2" is somewhat general thing (different for different machines).

File: README.md [hunk 2]
```diff
@@ -30,47 +30,47 @@ Supported systems are:
...
-   Change the line `exec "$DIR/Gothic2Notr" "$@"` to reflect your Gothic path e.g.
+   If OpenGothic fails to find your Gothic 2 files, you have to explicitly specify its location via `-g` parameter.
+   Change the line `exec "$DIR/Gothic2Notr" "$@"` to reflect your Gothic 2 path e.g.
 
    `exec "$DIR/Gothic2Notr" "$@" -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"`
 
 ---
 ### Modifications
-Mods can be installed as usual. Provide the `modfile.ini` to OpenGothic via `-game:` parameter to play. Example:
+Mods can be installed as usual. Provide the `modfile.ini` to OpenGothic via the `-game:` parameter to play. Example:
```
Comment (Try) [inline README.md:73]: > the `-game:`

similar to `the path`

**Review by Try — COMMENTED** [review]
Looks very good to me, thanks @LordKobra !

I've put a few questions on parts, that are not clear to me

File: README.md (hunk 1 above)
Comment (LordKobra) [reply README.md:52]: I'd argue on a semantic level it is a very specific path, as indicated by "to". From a developer perspective, it's also a unique path on an individual system.

File: README.md (hunk 2 above)
Comment (LordKobra) [reply README.md:73]: Here I'd argue again, the parameter is very specific: It is the `-game:` parameter, not any parameter.

**Comment by Try** [comment]
Thanks for clarifying about `the` :) 
Merged.

**Pull request merged**